shared/src/native-src/string.cpp

```diff
@@ -1,4 +1,5 @@
 #include "string.h"
+#include <algorithm>
```

Comment:
not a blocker, but it seems like you can revert the changes to both `string.cpp` and `string.h`